Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/Cellar/python@3.9/3.9.13/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import numpy as np
import matplotlib.pyplot as plt

def puissiter(A,v0,niter=5):
    v = v0
    vv = [v0]
    ll = [np.dot(v0,np.dot(A,v0))]
    for k in range(niter):
        w = np.dot(A,v)
        v = w/np.linalg.norm(w)
        lamda = np.dot(v,np.dot(A,v))
        vv.append(v)
        ll.append(lamda)
    return ll, vv

A = np.array([[2.,1,1],[1,3,1],[1,1,4]])
print("La plus grande valeur propre de A est ",lam =(np.linalg.eigvals(A)[0]))
ll, vv = puissiter(A,np.ones(3))

plt.plot(range(len(ll)),ll,'-o')
plt.plot(range(len(ll)),lam*np.ones((len(ll)), dtype=np.uint8) ,'r')
plt.ylabel('Puissances it√©r√©es')
plt.xlabel('Iteration');
plt.tight_layout()



------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Input [0;32mIn [1][0m, in [0;36m<module>[0;34m[0m
[1;32m     14[0m     [38;5;28;01mreturn[39;00m ll, vv
[1;32m     16[0m A [38;5;241m=[39m np[38;5;241m.[39marray([[[38;5;241m2.[39m,[38;5;241m1[39m,[38;5;241m1[39m],[[38;5;241m1[39m,[38;5;241m3[39m,[38;5;241m1[39m],[[38;5;241m1[39m,[38;5;241m1[39m,[38;5;241m4[39m]])
[0;32m---> 17[0m [38;5;28;43mprint[39;49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43mLa plus grande valeur propre de A est [39;49m[38;5;124;43m"[39;49m[43m,[49m[43mlam[49m[43m [49m[38;5;241;43m=[39;49m[43m([49m[43mnp[49m[38;5;241;43m.[39;49m[43mlinalg[49m[38;5;241;43m.[39;49m[43meigvals[49m[43m([49m[43mA[49m[43m)[49m[43m[[49m[38;5;241;43m0[39;49m[43m][49m[43m)[49m[43m)[49m
[1;32m     18[0m ll, vv [38;5;241m=[39m puissiter(A,np[38;5;241m.[39mones([38;5;241m3[39m))
[1;32m     20[0m plt[38;5;241m.[39mplot([38;5;28mrange[39m([38;5;28mlen[39m(ll)),ll,[38;5;124m'[39m[38;5;124m-o[39m[38;5;124m'[39m)

[0;31mTypeError[0m: 'lam' is an invalid keyword argument for print()
TypeError: 'lam' is an invalid keyword argument for print()

